CHANGES.rst

```diff
@@ -4,6 +4,10 @@ Change History
 2.5.3 (unreleased)
 ==================
 
+- In the tests, set ``TERM=dumb`` in the environment.  This avoids
+  invisible control characters popping up in some terminals, like
+  ``xterm``.  [maurits]
+
```

Comment:
Right, `system` is used in the tests of recipes, so others might be affected by this, so they need to know.
I will update the text.
I have run the `collective.recipe.backup` tests with this branch and they pass.